README.md

```diff
@@ -143,10 +143,10 @@ environment variables
   be ``SCIKIT_IMAGE_VERSION``). If specified it will override any version
   number limitations listed in ``CONDA_DEPENDENCIES``.
 
-* ``CONDA_CHANNEL_PRIORITY``: can be set to ``True`` or ``False``, and
+* ``CONDA_CHANNEL_PRIORITY``: can be set to ``strict``, ``flexible`` or ``disabled``, and
```

Comment:
what will happen to repos that used `True` and `False` before?

Reply:
`True` → `flexible`
`False` → `disabled`